src/getCoinCombination.test.js

```diff
@@ -7,4 +7,33 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
```

Comment:
I would suggest to use two more basic checks from your checklist:
https://mate-academy.github.io/qa-program/checklists/unit_tests.html#first-two-tests-should-validate-that-function-is-declared-and-data-type-the-function-returns